.env.example

```diff
@@ -21,3 +21,5 @@ SECRET_KEY=super_secret
 # machine user account for loader
 MACHINE_USER_LOADER_EMAIL=loader
 MACHINE_USER_LOADER_PASSWORD=password
+# machine user token has no expiry
+MACHINE_USER_LOADER_JWT=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJsb2FkZXIiLCJwZXJtaXNzaW9ucyI6ImFkbWluIn0.GRymxg-mbw2oO2zEBQm8p8eS7MqUSvBolljWgTufEcI
```

Comment:
Are you sure you want this key public to the world?

Reply:
Thanks for raising the concern! As this is just the example `.env` file, it should be OK.